packages/ui/src/core/check-box-group/check-box-group-with-icon.component.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```

Comment:
We shouldn't have a new component. We should add the ability to add the icon in the same checkbox component